README.md

```diff
@@ -1,5 +1,6 @@
 # The Algorithms - Python
-[![Gitpod Ready-to-Code](https://img.shields.io/badge/Gitpod-Ready--to--Code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/TheAlgorithms/Python) 
+[![Gitpod Ready-to-Code](https://img.shields.io/badge/Gitpod-Ready--to--Code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/TheAlgorithms/Python)
+![](https://img.shields.io/github/license/sourcerer-io/hall-of-fame.svg?colorB=ff0000)
```

Comment:
What is this?

Reply:
This is the license shield (MIT License) that I have added